tortoise/sim/generator.go

```diff
@@ -106,16 +107,16 @@ type Generator struct {
 	rng    *rand.Rand
 	conf   config
 
-	State State
+	states []State
 
 	nextLayer types.LayerID
 	// key is when to return => value is the layer to return
 	reordered   map[types.LayerID]types.LayerID
 	layers      []*types.Layer
+	units       [2]int
```

Comment:
what is the 1st and what is the 2nd element?

Reply:
lower and upper bound